src/Rebing/GraphQL/Support/SelectFields.php

```diff
@@ -173,7 +173,7 @@ protected static function handleFields(array $requestedFields, $parentType, arra
                             }
                         }
                         // If 'HasMany', then add it in the 'with'
-                        elseif ((is_a($relation, HasMany::class) || is_a($relation, MorphMany::class) || is_a($relation, HasOne::class) || is_a($relation, MorphOne::class))
+                        elseif ((is_a($relation, HasMany::class) || is_a($relation, MorphOne::class) || is_a($relation, MorphMany::class) || is_a($relation, HasOne::class) || is_a($relation, MorphOne::class))
```

Comment:
I don't understand this change, `MorphOne` _is already_ part of the `elseif` check (it's at the end), see the original line (emphasis mine):
>  elseif ((is_a($relation, HasMany::class) || is_a($relation, MorphMany::class) || is_a($relation, HasOne::class) || is_a($relation, **MorphOne::class**))

Can you elaborate how it fixes it and what exactly is broken with this change?